src/assets/fabricator/scripts/fabricator.js

```diff
@@ -96,64 +96,31 @@ fabricator.buildColorChips = function () {
 fabricator.setActiveItem = function () {
 
 	/**
-	 * @return {Array} Sorted array of menu item 'ids'
+	 * Match the window location with the menu item, set menu item as active
 	 */
-	var parsedItems = function () {
+	var setActive = function () {
 
-		var items = [],
-			id, href;
+		// get current file and hash without first slash
+		var current = (window.location.pathname + window.location.hash).replace(/^\//g, ''),
```

Comment:
This regex also needs to be modified to exclude trailing `.sub-component` in the url. For instance, in the default Fabricator instance when you navigate to http://localhost:3000/components.html#lists.ordered the "Lists" item in the side bar should highlight.